shared/src/main/java/com/alliander/osgp/shared/infra/ws/WebServiceTemplateFactory.java

```
@@ -46,7 +47,7 @@
 
     private final Jaxb2Marshaller marshaller;
     private final SaajSoapMessageFactory messageFactory;
-    private final String defaultUri;
+    private String defaultUri;
```

I think that the name 'defaultUri' may be misleading, in light of the 'responseUrl'  story. I discussed about this with Joep, and we both agreed that: 'targetUri' is a better name.